team-telnyx/webrtc
# Refactor Storybook react-audio examples

Refactors react-audio audio examples to use globally configured authentication.

Also,
- Removed `environment`, since this config should not be end-user facing
- Added `token` authentication option
- Addresses password being exposed with storybook-knobs usage

This is in preparation for consolidating with react-video examples.

## 📝 To Do


## ✋ Manual testing

1. [`packages/js/examples/react-audio`]: `npm install`
2. [`packages/js/examples/react-audio`]: `npm run storybook`
3. Go to http://localhost:6006/?path=/story/audio--click-to-call and enter in your credentials. Verify both SIP connection creds and token work
4. Go to http://localhost:6006/?path=/story/audio--web-dialer and click "Call". Verify calling works

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

<img width="931" alt="Screen Shot 2021-01-19 at 1 58 16 PM" src="https://user-images.githubusercontent.com/4672952/105098155-82a1ce80-5a5e-11eb-980f-c87e2f73dc55.png">

<img width="717" alt="Screen Shot 2021-01-19 at 1 58 41 PM" src="https://user-images.githubusercontent.com/4672952/105098182-8afa0980-5a5e-11eb-8077-026486214f05.png">
